go.mod

```diff
@@ -233,6 +233,12 @@ require (
 	software.sslmate.com/src/go-pkcs12 v0.5.0
 )
 
+require (
+	github.com/aws/aws-sdk-go-v2/service/identitystore v1.27.3
+	github.com/aws/aws-sdk-go-v2/service/organizations v1.34.3
+	github.com/aws/aws-sdk-go-v2/service/ssoadmin v1.29.3
+)
```

Comment:
Can we move this block to be near the other aws sdk deps?

Reply:
https://github.com/gravitational/teleport/pull/48192/commits/73a71ef8520e4f100c840fd9e72b30ff3732ef71